github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```
@@ -323,3 +323,44 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      timeout: 0h10m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+    max_concurrency: 1
+    name: check-provision-alpine-container-disk
```

Can you rename the job to make it more clear? There's already an alpine container disk in existence.
```suggestion
    name: check-provision-alpine-cloud-init-container-disk
```

---

using "with-test-tooling" we have a similar one for fedora